specification/workloads/Workloads.SAPVirtualInstance.Management/tspconfig.yaml

```diff
@@ -13,6 +13,12 @@ options:
     # `arm-resource-flattening` is only used for back-compat for specs existed on July 2024. All new service spec should NOT use this flag
     arm-resource-flattening: true
     output-file: "{azure-resource-provider-folder}/{service-name}/SAPVirtualInstance/{version-status}/{version}/SAPVirtualInstance.json"
+  # "@azure-tools/typespec-csharp":
+  #   flavor: azure
+  #   package-dir: "Azure.ResourceManager.SapVirtualInstances"
+  #   clear-output-folder: true
+  #   model-namespace: false
+  #   namespace: "{package-dir}"
```

Comment:
Is this expected, we are commenting out c# sdk generation.

Reply:
For C# SDK, we are facing some generation build failures with Dotnet SDK, which we choose to pick after fixing the current issues